vignettes/customise.Rmd

```diff
@@ -226,23 +226,26 @@ It can contain any of the following:
 -   A configuration file in `inst/pkgdown/_pkgdown.yml`. This can be used to set (e.g.) author definitions, Bootstrap version and variables, the sidebar, footer, navbar, etc.
 -   Templates in `inst/pkgdown/templates/` will override the default templates.
 -   Assets in `inst/pkgdown/assets/` will be copied in to the destination directory.
+    (Note this files are only copied; you'll need to reference them in your stylesheet or elsewhere in order for them to be actually used.)
 -   `inst/pkgdown/extra.scss` will be added to the bslib ruleset.
+    (Note that `extra.css` is not supported in templates.)
 
-Any configuration/files supplied will override the pkgdown defaults, but will be overridden by site specific settings.
+The pkgdown defaults will be overriden by these template files, which are in turn overridden by site specific settings.
```

Comment:
```suggestion
The pkgdown defaults will be overriden by these template files, which are in turn overridden by package specific settings.
```

Although package is confusing too as the template package is a package. :sweat_smile: "package under documentation"